eoscompanion/main.py

```diff
@@ -157,6 +157,31 @@ def do_activate(self):  # pylint: disable=arguments-differ
         logging.info('Activated')
         return Gio.Application.do_activate(self)
 
+CONFIG_FILE = '/etc/eos-companion-app/config.ini'
+CONFIG_SECTION = 'Companion App'
+CONFIG_NAME = 'loglevel'
+DEFAULT_LOG_LEVEL = logging.INFO
+
+def getLogLevel():
+    keyfile = GLib.KeyFile()
+    try:
+        keyfile.load_from_file(CONFIG_FILE, GLib.KeyFileFlags.NONE)
+    except GLib.Error:
+        return DEFAULT_LOG_LEVEL
+
+    try:
+        configLevel = keyfile.get_value(CONFIG_SECTION, CONFIG_NAME)
+    except GLib.Error:
```

Comment:
It is probably worth using `except GLib.Error as error: ... if error.matches` to filter for the errors that we will treat as non-fatal. I think those would be `G_KEY_FILE_ERROR_KEY_NOT_FOUND` `G_KEY_FILE_ERROR_GROUP_NOT_FOUND` and `G_KEY_FILE_ERROR_INVALID_VALUE`

Reply:
Still need to match the error here